src/nwb2bids/bids_models/_probes.py

```diff
@@ -111,5 +111,8 @@ def to_json(self, file_path: str | pathlib.Path) -> None:
         file_path : path
             The path to the output JSON file.
         """
+        if isinstance(file_path, str):
+            file_path = pathlib.Path(file_path)
```

Comment:
```suggestion
def to_json(self, file_path: pydantic.FilePath) -> None:
"""
       Save the probes information to a JSON file.

       Parameters
       ----------
       file_path : path
           The path to the output JSON file.
       """
```

Looks like this should work then

Also has benefit of asserting if file input exists/resolves on the system